jujugui/static/gui/src/app/components/env-switcher/list/list.js

```diff
@@ -120,29 +142,28 @@ YUI.add('env-list', function() {
 
     render: function() {
       const auth = this.props.authDetails;
-      let buttonRow;
+      let createNew;
       if (auth && auth.rootUserName) {
```

Comment:
Not sure whether having the create new hidden behind auth is required? @hatched @frankban?